fairlearn/reductions/_moments/utility_parity.py

```diff
@@ -6,7 +6,8 @@
 
 from .moment import ClassificationMoment
 from .moment import _GROUP_ID, _LABEL, _PREDICTION, _ALL, _EVENT, _SIGN
-from fairlearn._input_validation import _MESSAGE_RATIO_NOT_IN_RANGE, _validate_and_reformat_input
+from fairlearn.utils._input_validation import _MESSAGE_RATIO_NOT_IN_RANGE, \
+    _validate_and_reformat_input
```

Comment:
```suggestion
from fairlearn.utils._input_validation import (
    _MESSAGE_RATIO_NOT_IN_RANGE,
    _validate_and_reformat_input
)
```

Or even two separate imports :D